app/collections/collections-manager.jsx

```diff
@@ -1,46 +1,80 @@
 import React from 'react';
 import CollectionsCreateForm from './create-form';
 import CollectionSearch from './collection-search';
+import LoadingIndicator from '../components/loading-indicator';
 
 class CollectionsManager extends React.Component {
   constructor(props) {
     super(props);
 
     this.state = {
+      adding: false,
       collections: [],
-      error: null,
-      hasCollectionSelected: false
+      errors: [],
+      hasCollectionSelected: true
     };
 
     this.addToCollections = this.addToCollections.bind(this);
+    this.onChange = this.onChange.bind(this);
+  }
+
+  onChange() {
+    const collections = this.search.getSelected();
+
+    if (collections.length > 0) {
+      this.setState({ hasCollectionSelected: false });
+    }
```

Comment:
Right. I'll reverse the logic since it's being used to manage the button disabled state.